src/mono/wasm/runtime/polyfills.ts

```diff
@@ -0,0 +1,50 @@
+import { ENVIRONMENT_IS_NODE, Module } from "./imports";
+
+export async function fetch_like(url: string): Promise<Response> {
+    try {
+        if (typeof (globalThis.fetch) === "function") {
+            return globalThis.fetch(url, { credentials: "same-origin" });
```

Comment:
Is it correct to throw away original fetch settings?

Reply:
I just moved the method from `_fetch_asset` in `startup.ts` ... but I think it's ok.
When emscripten is calling this, they are sending the same options. 
It's also the default behavior of fetch. See https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials

Maybe it's there for purpose, so that it could not be used with `include` ?